### PR TITLE
main/fcitx5: add bash dependency

### PR DESCRIPTION
**Pull request by denisstrizhkin**
`fcitx5-diagnose` requires bash to work because it's a bash script

### CODE REVIEW

**Comment by q66** [comment]
if it's not mandatory for function it should be split in a separate package because pulling in bash through library dependencies is bad

**Comment by nekopsykose** [comment]
fcitx5 isn't a library dep, it's a whole ime you have to manually install and is never present otherwise

should still be split anyway tho

**Comment by denisstrizhkin** [comment]
As in make fcitx5-diagnose a separate package?

File: main/fcitx5/template.py [hunk 1]
```diff
@@ -56,3 +56,9 @@ def post_extract(self):
 @subpackage("fcitx5-devel")
 def _(self):
     return self.default_devel()
+
+
+@subpackage("fcitx5-diagnose")
+def _(self):
+    self.depends = ["bash"]
```
Comment (nekopsykose) [inline main/fcitx5/template.py:63]:
```suggestion
    self.depends = [self.parent, "bash"]
    self.install_if = [self.parent, "bash"]
```

**Pull request merged**